configure.ac

```diff
@@ -738,28 +724,21 @@ AS_IF([   test $have_ch_image = yes \
       [have_ch_image_rsync=yes],
       [have_ch_image_rsync=no])
 
-AS_IF([   test -n "$DOCKER" \
-       && test -n "$MKTEMP"],
```

Comment:
Can we remove the `$MKTEMP` tests too? I have a recollection that only Docker uses it?